### DISCUSSION TITLE
v1.2.0

### DISCUSSION THREAD

**Discussion by mattinannt** [formbricks/formbricks#1466] [Announcements]
Formbricks v.1.2.0 is here 🎉
Wow, what a great ride through our first Formtribe Hackathon. Thank you so much for all the amazing contributions! 🚀
We have some great new features in this release as well as major security improvements (including 2 factor authentication) as well as new integrations. Be sure to check out our [Migration Guide](https://formbricks.com/docs/self-hosting/migration-guide) for this release.

A big thank you to our amazing contributors who allowed us to add so many new features and bug fixes in such a short amount of time. 🎉🙏🚀

## New features & improvements 🚀
* feat: 2FA by @pandeymangg in https://github.com/formbricks/formbricks/pull/798
* feat: Google sheet integration by @Dhruwang in https://github.com/formbricks/formbricks/pull/1128
* feat: Add Animated Background Files by @jobenjada in https://github.com/formbricks/formbricks/pull/1441
* feat: Add source to metadata submission by @gupta-piyush19 in https://github.com/formbricks/formbricks/pull/1323
* feat: Add welcome card by @KapadiaNaitik in https://github.com/formbricks/formbricks/pull/1073
* feat: Added Dynamic Link Previews by @anjy7 in https://github.com/formbricks/formbricks/pull/1093
* feat: Added Pin Protection to forms by @piyushgarg-dev in https://github.com/formbricks/formbricks/pull/1142
* feat: Release @formbricks/js@1.1.3 by @ShubhamPalriwala in https://github.com/formbricks/formbricks/pull/1379
* feat: Storybook integration for UI components  by @anjy7 in https://github.com/formbricks/formbricks/pull/987
* feat: add SSO login via Azure AD (second try) by @jonas-hoebenreich in https://github.com/formbricks/formbricks/pull/1188
* feat: add airtable integration by @G3root in https://github.com/formbricks/formbricks/pull/926
* feat: add docs for setting ENCRYPTION_KEY in .env in contributing guide by @ShubhamPalriwala in https://github.com/formbricks/formbricks/pull/1088
* feat: added Firecamp to oss-friends the list by @shreya-gr in https://github.com/formbricks/formbricks/pull/1291
* feat: added dropoff visibility by @gupta-piyush19 in https://github.com/formbricks/formbricks/pull/907
* feat: adds hidden field functionality to surveys by @rtpa25 in https://github.com/formbricks/formbricks/pull/1144
* feat: adds image adding option to all questions by @rtpa25 in https://github.com/formbricks/formbricks/pull/1305
* feat: authorization in attributeClassAction by @ShubhamPalriwala in https://github.com/formbricks/formbricks/pull/1056
* feat: authorization in deletePerson action by @ShubhamPalriwala in https://github.com/formbricks/formbricks/pull/1063
* feat: authorization in notificationUpdateAction by @ShubhamPalriwala in https://github.com/formbricks/formbricks/pull/1058
* feat: authorization in product update in look and feel page by @ShubhamPalriwala in https://github.com/formbricks/formbricks/pull/1060
* feat: authorization in responseNote actions & cleanup of unused actions by @ShubhamPalriwala in https://github.com/formbricks/formbricks/pull/1062
* feat: brand color overwrite by @gupta-piyush19 in https://github.com/formbricks/formbricks/pull/1035
* feat: client side pagination in open text questions in responses summary by @ShubhamPalriwala in https://github.com/formbricks/formbricks/pull/1422
* feat: formbricks-js 1.1.4 release by @mattinannt in https://github.com/formbricks/formbricks/pull/1455
* feat: introduce new security policy by @ShubhamPalriwala in https://github.com/formbricks/formbricks/pull/1033
* feat: lazy loading and endless scrolling for responses page by @ShubhamPalriwala in https://github.com/formbricks/formbricks/pull/1421
* feat: let's v1.2 by @ShubhamPalriwala in https://github.com/formbricks/formbricks/pull/1332
* feat: move formbricks-js to vite bundler by @mattinannt in https://github.com/formbricks/formbricks/pull/883
* feat: rate limiting of auth endpoints by @ShubhamPalriwala in https://github.com/formbricks/formbricks/pull/1227
* feat: release patch release formbricks/js with changesets by @ShubhamPalriwala in https://github.com/formbricks/formbricks/pull/1376
* feat: update formtribe leaderboard as per today by @ShubhamPalriwala in https://github.com/formbricks/formbricks/pull/1310
* feat: update latest leaderboard points by @ShubhamPalriwala in https://github.com/formbricks/formbricks/pull/1247
* feat: update leaderboard as per today by @ShubhamPalriwala in https://github.com/formbricks/formbricks/pull/1408
* feat: update leaderboard as per today by @ShubhamPalriwala in https://github.com/formbricks/formbricks/pull/1457
* feat: use formbricks/api package internally in /js & /lib by @ShubhamPalriwala in https://github.com/formbricks/formbricks/pull/955

## Bugfixes 🐛
* fix Add action UI issue on mobile screen #1254 by @bilalmirza74 in https://github.com/formbricks/formbricks/pull/1306
* fix: fixed1296 href link switch by @adityadeshlahre in https://github.com/formbricks/formbricks/pull/1317
* fix: removed user from Team, but user can select team by @adityadeshlahre in https://github.com/formbricks/formbricks/pull/1111
* fix: fix horizontal scrollbar in small viewports by @Bitnagar in https://github.com/formbricks/formbricks/pull/1100
* fix: Fix user dropdown keyboard accessibility by @Bitnagar in https://github.com/formbricks/formbricks/pull/1240
* fix: Typo-main-page by @asharan2511 in https://github.com/formbricks/formbricks/pull/1146
* fix:  CODE_OF_CONDUCT.md by @PentesterPriyanshu in https://github.com/formbricks/formbricks/pull/1390
* fix: #1353 validating pin before publishing by @vikaspatil0021 in https://github.com/formbricks/formbricks/pull/1356
* fix: #995 Multi select Other option selected automatically when come… by @rohan9896 in https://github.com/formbricks/formbricks/pull/1004
* fix: Added DISCORD to the Footer.tsx by @Sha1kh4 in https://github.com/formbricks/formbricks/pull/1154
* fix: Air table callback by @Dhruwang in https://github.com/formbricks/formbricks/pull/1418
* fix: Alignment of step-by-step and try button for mobile by @NilanjanPramanik in https://github.com/formbricks/formbricks/pull/1241
* fix: Automate copyright year by @aryabyte21 in https://github.com/formbricks/formbricks/pull/1136
* fix: Button Enablement Precedes Component Rendering on Publish/Save Click by @Shyam-Raghuwanshi in https://github.com/formbricks/formbricks/pull/1400
* fix: Buttons Margin and code snippets issue across Docs by @bilalmirza74 in https://github.com/formbricks/formbricks/pull/1107
* fix: Concierge btn text opacity fixed and made it look proper by @ratishjain12 in https://github.com/formbricks/formbricks/pull/1162
* fix: Copy button disabled while it's value is null by @Shyam-Raghuwanshi in https://github.com/formbricks/formbricks/pull/1328
* fix: Corrected landing page styles by @vineetjk in https://github.com/formbricks/formbricks/pull/1278
* fix: Create dark mode version of Code Block on landing page by @itzabhinavarya in https://github.com/formbricks/formbricks/pull/1358
* fix: Data is not being sent webhook api on trigger (#1285) by @rohan9896 in https://github.com/formbricks/formbricks/pull/1316
* fix: Display appropriate error message when uploading image by @Olaleye-Blessing in https://github.com/formbricks/formbricks/pull/1334
* fix: Exit Intent not working on FF, Safari, Arc by @prasoonmhwr in https://github.com/formbricks/formbricks/pull/1059
* fix: Fixed grammatical error in feedback box -> why it is useful section by @3xp10it3r in https://github.com/formbricks/formbricks/pull/1166
* fix: In Review Prompt Lower and Upper labels are not showing in dummyUi/Preview by @itzabhinavarya in https://github.com/formbricks/formbricks/pull/1402
* fix: Link for "Pricing" not working fixed by @sohamtembhurne in https://github.com/formbricks/formbricks/pull/1165
* fix: Logo is not visible in edge search #1265 by @bilalmirza74 in https://github.com/formbricks/formbricks/pull/1330
* fix: Name can updated without any characters/value by @adityadeshlahre in https://github.com/formbricks/formbricks/pull/1020
* fix: Overflow of body across leaderboard on mobile screen #1120 & Doc's navbar was not responsive #1129 by @bilalmirza74 in https://github.com/formbricks/formbricks/pull/1123
* fix: Page break on Convert to Multi Select by @ruhani2 in https://github.com/formbricks/formbricks/pull/1383
* fix: Pricing Link in mobile menu on LP , Update Header.tsx by @sundaram2021 in https://github.com/formbricks/formbricks/pull/1163
* fix: Refactor theme switcher from dropdown to toggle by @moheyt in https://github.com/formbricks/formbricks/pull/1360
* fix: Resolve issue with "Back to Home" button on 404 page by @Shyam-Raghuwanshi in https://github.com/formbricks/formbricks/pull/1381
* fix: Share modal not opening by @hemantwasthere in https://github.com/formbricks/formbricks/pull/1387
* fix: Survey : Welcome Card Button is buggy by @itzabhinavarya in https://github.com/formbricks/formbricks/pull/1299
* fix: Typo on main page is fixed by @vedant-z in https://github.com/formbricks/formbricks/pull/1250
* fix: Unifying discord logo by @moheyt in https://github.com/formbricks/formbricks/pull/1236
* fix: Validate Product Names by @adityadeshlahre in https://github.com/formbricks/formbricks/pull/1362
* fix: Welcome card not being active by default in every template by @RajuGangitla in https://github.com/formbricks/formbricks/pull/1372
* fix: X and optional text overlap by @AnushDeokar in https://github.com/formbricks/formbricks/pull/1077
* fix: add toast and error handler for multiple team/product creation with existing team/product name by @anjy7 in https://github.com/formbricks/formbricks/pull/996
* fix: add twitter title, desc and keyword meta tags by @aashishpanthi in https://github.com/formbricks/formbricks/pull/1359
* fix: added script to setup encryption key in gitpod script by @anjy7 in https://github.com/formbricks/formbricks/pull/1080
* fix: added scroll to top button by @impolska742 in https://github.com/formbricks/formbricks/pull/1153
* fix: adding loader for Delete and Save buttons by @Shyam-Raghuwanshi in https://github.com/formbricks/formbricks/pull/1344
* fix: attribute-settings-tab-button-link by @3xp10it3r in https://github.com/formbricks/formbricks/pull/1221
* fix: build issues in environment service on some systems by @mattinannt in https://github.com/formbricks/formbricks/pull/1160
* fix: cast .length result to boolean by @rotimi-best in https://github.com/formbricks/formbricks/pull/1253
* fix: changes env names by @pandeymangg in https://github.com/formbricks/formbricks/pull/1259
* fix: concierge cards by @ratishjain12 in https://github.com/formbricks/formbricks/pull/1263
* fix: conflicting typescript and dts from surveys package producing error by @mattinannt in https://github.com/formbricks/formbricks/pull/1143
* fix: dependency reference issues in monorepo by @mattinannt in https://github.com/formbricks/formbricks/pull/1141
* fix: docker compose volume only takes absolute paths by @ShubhamPalriwala in https://github.com/formbricks/formbricks/pull/1206
* fix: duplicate survey not revalidating the cache key PAIR: Bilal Mirza by @ortin779 in https://github.com/formbricks/formbricks/pull/1229
* fix: email verification before answering survey by @Dhruwang in https://github.com/formbricks/formbricks/pull/1460
* fix: encodes url correctly by @pandeymangg in https://github.com/formbricks/formbricks/pull/1252
* fix: file uploads api changes by @pandeymangg in https://github.com/formbricks/formbricks/pull/1319
* fix: fix invalid state initialisation by @ortin779 in https://github.com/formbricks/formbricks/pull/1220
* fix: fixes client storage api route method by @pandeymangg in https://github.com/formbricks/formbricks/pull/1451
* fix: fixing navbar position while scrolling the page. by @itzabhinavarya in https://github.com/formbricks/formbricks/pull/1266
* fix: format survey date fields back to Date type by @rotimi-best in https://github.com/formbricks/formbricks/pull/1096
* fix: formbricks-js not starting in a browser by @mattinannt in https://github.com/formbricks/formbricks/pull/1131
* fix: formbricks-surveys not compiling correctly leads to formbricks-js throwing error in action creation by @mattinannt in https://github.com/formbricks/formbricks/pull/1223
* fix: formtribe leaderboard name correction by @ShubhamPalriwala in https://github.com/formbricks/formbricks/pull/1313
* fix: getAttributeClassByName cache returns wrong entries by @mattinannt in https://github.com/formbricks/formbricks/pull/1072
* fix: handle display with no response data in per question drop off by @ShubhamPalriwala in https://github.com/formbricks/formbricks/pull/1374
* fix: have same non-module workflow for packages by @mattinannt in https://github.com/formbricks/formbricks/pull/1230
* fix: hide email input in survey share modal when share code is displayed by @mattinannt in https://github.com/formbricks/formbricks/pull/1432
* fix: icons not aligned center on firefox by @AliYar-Khan in https://github.com/formbricks/formbricks/pull/1251
* fix: image and page responsiveness in docs section by @itzabhinavarya in https://github.com/formbricks/formbricks/pull/1078
* fix: make metadata internal fields optionality uniform in response object by @ShubhamPalriwala in https://github.com/formbricks/formbricks/pull/1373
* fix: minor ui fixes by @aryabyte21 in https://github.com/formbricks/formbricks/pull/1138
* fix: multiChoice undefined value check by @Dhruwang in https://github.com/formbricks/formbricks/pull/1087
* fix: names spellings in survey pin related functions or variables by @yatharth1706 in https://github.com/formbricks/formbricks/pull/1232
* fix: pre-commit-hook by @rtpa25 in https://github.com/formbricks/formbricks/pull/1203
* fix: prefiling in welcome card surveys by @gupta-piyush19 in https://github.com/formbricks/formbricks/pull/1352
* fix: prevent exit intent and scroll intent in js package to make api call by @ShubhamPalriwala in https://github.com/formbricks/formbricks/pull/1350
* fix: rate limit is now enabled on node prod envs only by @ShubhamPalriwala in https://github.com/formbricks/formbricks/pull/1424
* fix: remove prisma types import in runtime by @mattinannt in https://github.com/formbricks/formbricks/pull/1228
* fix: replace ' with backticks by @jobenjada in https://github.com/formbricks/formbricks/pull/1214
* fix: responses not visible when sdk not connected by @mattinannt in https://github.com/formbricks/formbricks/pull/1074
* fix: scrollbar issue by @Dhruwang in https://github.com/formbricks/formbricks/pull/1090
* fix: statusText mismatch in webhook integrations by @SiR-PENt in https://github.com/formbricks/formbricks/pull/1068
* fix: survey baseurl fix and addition of SHORT_SURVEY_BASE_URL in .env.example file by @rohan9896 in https://github.com/formbricks/formbricks/pull/1046
* fix: survey editor minor css bug fixes by @ShubhamPalriwala in https://github.com/formbricks/formbricks/pull/1287
* fix: survey list loading error for products with a lot of actions by @mattinannt in https://github.com/formbricks/formbricks/pull/1302
* fix: survey trigger cannot be deleted by @mattinannt in https://github.com/formbricks/formbricks/pull/1271
* fix: survey url incorrect in single use share dialoge by @mattinannt in https://github.com/formbricks/formbricks/pull/1464
* fix: survey with empty pin cannot be created by @mattinannt in https://github.com/formbricks/formbricks/pull/1361
* fix: the background does not load up on the entire page!  by @Shyam-Raghuwanshi in https://github.com/formbricks/formbricks/pull/1331
* fix: toggle button text when switching between light and dark mode by @Shyam-Raghuwanshi in https://github.com/formbricks/formbricks/pull/1309
* fix: triggered survey not behaving correctly after userId change by @mattinannt in https://github.com/formbricks/formbricks/pull/1453
* fix: twitter-icon-in-docs-footer-section by @itzabhinavarya in https://github.com/formbricks/formbricks/pull/1134
* fix: type errors in js sdk by @G3root in https://github.com/formbricks/formbricks/pull/1115
* fix: typo in gdpr-guide.mdx by @eltociear in https://github.com/formbricks/formbricks/pull/1043
* fix: update product name, waiting time and delete product operation by @3xp10it3r in https://github.com/formbricks/formbricks/pull/1260
* fix: uploads by @pandeymangg in https://github.com/formbricks/formbricks/pull/1449
* fix: use ENCRYPTION_KEY in single use surveys by @pandeymangg in https://github.com/formbricks/formbricks/pull/1094
* fix: use strings for pin datatype to allow more variations by @mattinannt in https://github.com/formbricks/formbricks/pull/1335
* fix: validate cssSelector value by @ortin779 in https://github.com/formbricks/formbricks/pull/1420
* fix: web-app surveys not triggered with modified api endpoint by @mattinannt in https://github.com/formbricks/formbricks/pull/1097

## Chores & Code improvements 🤓
* chore: bump webpack from 5.75.0 to 5.88.2 by @dependabot in https://github.com/formbricks/formbricks/pull/1121
* chore: add GitHub trending badge in readme file by @liweiyi88 in https://github.com/formbricks/formbricks/pull/1279
* chore: FOR-1430 Component relocation by @neilchauhan2 in https://github.com/formbricks/formbricks/pull/1261
* chore: Move Billing API to App directory & Deprecate unused /pages/api/[environment|team] endpoints by @ShubhamPalriwala in https://github.com/formbricks/formbricks/pull/1423
* chore: Remove SurveyWithAnalytics model to simplify survey handling by @mattinannt in https://github.com/formbricks/formbricks/pull/1273
* chore: Remove response limit in preparation for new pricing by @mattinannt in https://github.com/formbricks/formbricks/pull/1443
* chore: Update README.md by @rohitd2103 in https://github.com/formbricks/formbricks/pull/1024
* chore: Update npm dependencies by @mattinannt in https://github.com/formbricks/formbricks/pull/1242
* chore: Version Packages by @mattinannt in https://github.com/formbricks/formbricks/pull/1126
* chore: add ENCRYPTION_KEY to github actions by @mattinannt in https://github.com/formbricks/formbricks/pull/1069
* chore: add cla to PR template & /cla by @jobenjada in https://github.com/formbricks/formbricks/pull/1351
* chore: add dyrector.io by @gerimate in https://github.com/formbricks/formbricks/pull/1040
* chore: add folder structure for v2 components by @mattinannt in https://github.com/formbricks/formbricks/pull/1314
* chore: add hyperlink SECURITY reference by @madlia in https://github.com/formbricks/formbricks/pull/1245
* chore: add hyperlink to LICENSE reference by @malivinayak in https://github.com/formbricks/formbricks/pull/1246
* chore: add nodejs URL in readme by @krishvsoni in https://github.com/formbricks/formbricks/pull/1102
* chore: caching activity, display and person service by @rotimi-best in https://github.com/formbricks/formbricks/pull/1272
* chore: caching for action and actionClass services by @rotimi-best in https://github.com/formbricks/formbricks/pull/1264
* chore: caching response service by @rotimi-best in https://github.com/formbricks/formbricks/pull/1108
* chore: improve error component and display message to user by @mattinannt in https://github.com/formbricks/formbricks/pull/1298
* chore: improve error logging in client endpoints by @mattinannt in https://github.com/formbricks/formbricks/pull/1445
* chore: improve error logging in services by @mattinannt in https://github.com/formbricks/formbricks/pull/1300
* chore: improve logging in survey service by @mattinannt in https://github.com/formbricks/formbricks/pull/1156
* chore: include new features in pricing table by @mattinannt in https://github.com/formbricks/formbricks/pull/1438
* chore: increase MTU limit in Formbricks Cloud by @mattinannt in https://github.com/formbricks/formbricks/pull/1456
* chore: move group components in app folder by @rotimi-best in https://github.com/formbricks/formbricks/pull/1133
* chore: refactor services with Formbricks best practices by @rotimi-best in https://github.com/formbricks/formbricks/pull/988
* chore: refactor sync endpoint to use existing services by @mattinannt in https://github.com/formbricks/formbricks/pull/1417
* chore: remove build from source deployment guide by @ShubhamPalriwala in https://github.com/formbricks/formbricks/pull/1207
* chore: remove swr from dependencies by @mattinannt in https://github.com/formbricks/formbricks/pull/1407
* chore: simplify packages dependencies by @mattinannt in https://github.com/formbricks/formbricks/pull/1243
* chore: update github build action by @mattinannt in https://github.com/formbricks/formbricks/pull/1065
* chore: update turbo by @neilchauhan2 in https://github.com/formbricks/formbricks/pull/1345
* chore: Console tokenRes by @Dhruwang in https://github.com/formbricks/formbricks/pull/1434
* chore: Move all @formbricks/ui components to the new folder structure by @Dhruwang in https://github.com/formbricks/formbricks/pull/1057
* chore: Unified single response card and improved UX by @Dhruwang in https://github.com/formbricks/formbricks/pull/750
* chore: removes old types by @Dhruwang in https://github.com/formbricks/formbricks/pull/1288
* chore: #1343 Added Jost Web fonts by @SumanKisku in https://github.com/formbricks/formbricks/pull/1346
* chore: Fixed a minor grammatical mistake. by @Sha1kh4 in https://github.com/formbricks/formbricks/pull/1164

## Documentation Updates 📕
* docs: Added a 'Back to Top" button by @mishrasamiksha in https://github.com/formbricks/formbricks/pull/1318
* docs: updated CONTRIBUTING.md by @0Armaan025 in https://github.com/formbricks/formbricks/pull/1394
* docs: code quality & styling section by @rotimi-best in https://github.com/formbricks/formbricks/pull/1226
* docs: fix typo in apps/formbricks-com/app/docs/actions/code/page.mdx by @suravkshrestha in https://github.com/formbricks/formbricks/pull/1321

## New Contributors
* @rohitd2103 made their first contribution in https://github.com/formbricks/formbricks/pull/1024
* @gerimate made their first contribution in https://github.com/formbricks/formbricks/pull/1040
* @SiR-PENt made their first contribution in https://github.com/formbricks/formbricks/pull/1068
* @creator0131 made their first contribution in https://github.com/formbricks/formbricks/pull/1075
* @AnushDeokar made their first contribution in https://github.com/formbricks/formbricks/pull/1077
* @krishvsoni made their first contribution in https://github.com/formbricks/formbricks/pull/1102
* @Bitnagar made their first contribution in https://github.com/formbricks/formbricks/pull/1100
* @bilalmirza74 made their first contribution in https://github.com/formbricks/formbricks/pull/1107
* @impolska742 made their first contribution in https://github.com/formbricks/formbricks/pull/1153
* @DonKoko made their first contribution in https://github.com/formbricks/formbricks/pull/1110
* @Sha1kh4 made their first contribution in https://github.com/formbricks/formbricks/pull/1154
* @asharan2511 made their first contribution in https://github.com/formbricks/formbricks/pull/1146
* @aryabyte21 made their first contribution in https://github.com/formbricks/formbricks/pull/1136
* @sundaram2021 made their first contribution in https://github.com/formbricks/formbricks/pull/1163
* @3xp10it3r made their first contribution in https://github.com/formbricks/formbricks/pull/1166
* @rtpa25 made their first contribution in https://github.com/formbricks/formbricks/pull/1144
* @piyushgarg-dev made their first contribution in https://github.com/formbricks/formbricks/pull/1142
* @ortin779 made their first contribution in https://github.com/formbricks/formbricks/pull/1220
* @madlia made their first contribution in https://github.com/formbricks/formbricks/pull/1245
* @malivinayak made their first contribution in https://github.com/formbricks/formbricks/pull/1246
* @moheyt made their first contribution in https://github.com/formbricks/formbricks/pull/1236
* @vedant-z made their first contribution in https://github.com/formbricks/formbricks/pull/1250
* @AliYar-Khan made their first contribution in https://github.com/formbricks/formbricks/pull/1251
* @NilanjanPramanik made their first contribution in https://github.com/formbricks/formbricks/pull/1241
* @vineetjk made their first contribution in https://github.com/formbricks/formbricks/pull/1278
* @shreya-gr made their first contribution in https://github.com/formbricks/formbricks/pull/1291
* @liweiyi88 made their first contribution in https://github.com/formbricks/formbricks/pull/1279
* @Shyam-Raghuwanshi made their first contribution in https://github.com/formbricks/formbricks/pull/1309
* @suravkshrestha made their first contribution in https://github.com/formbricks/formbricks/pull/1321
* @mishrasamiksha made their first contribution in https://github.com/formbricks/formbricks/pull/1318
* @vikaspatil0021 made their first contribution in https://github.com/formbricks/formbricks/pull/1356
* @aashishpanthi made their first contribution in https://github.com/formbricks/formbricks/pull/1359
* @RajuGangitla made their first contribution in https://github.com/formbricks/formbricks/pull/1372
* @ruhani2 made their first contribution in https://github.com/formbricks/formbricks/pull/1383
* @hemantwasthere made their first contribution in https://github.com/formbricks/formbricks/pull/1387
* @SumanKisku made their first contribution in https://github.com/formbricks/formbricks/pull/1346
* @Olaleye-Blessing made their first contribution in https://github.com/formbricks/formbricks/pull/1334
* @0Armaan025 made their first contribution in https://github.com/formbricks/formbricks/pull/1394
* @PentesterPriyanshu made their first contribution in https://github.com/formbricks/formbricks/pull/1390

**Full Changelog**: https://github.com/formbricks/formbricks/compare/v1.1.2...v1.2.0

<hr /><em>This discussion was created from the release <a href='https://github.com/formbricks/formbricks/releases/tag/v1.2.0'>v1.2.0</a>.</em>

**Comment by bilalmirza74** [comment]
Congratulations for Brand New Release everyone

**Comment by hemantwasthere** [comment]
Congratulations 🚀🚀🔥

**Comment by prasoonmhwr** [comment]
Way to go

**Comment by impolska742** [comment]
My first open-source contribution! LFG! Congratulations everyone! 🚀 🔥

**Comment by Shyam-Raghuwanshi** [comment]
Congratulations to all my friends 💐
